lib/heroku.rb

```diff
@@ -0,0 +1,13 @@
+class Heroku
+  def self.create_users(organisation_id)
+    organisation = Content::Item.find(organisation_id)
+
+    User.create!(
+      name: 'AT user',
+      email: 'uat@gds.com',
+      uid: SecureRandom.uuid,
```

Comment:
Wouldn't this eventually fail in the (very rare) situation in that the same uuid is generated twice?
We could add a loop to check if a uuid was already generated. I mean something like:
```
def generate_uid
  loop do
     random_uid = SecureRandom.urlsafe_base64(nil, false)
     break random_uid unless User.exists?(uid: random_uid)
   end
 end
```

Reply:
@mgrassotti ok, I will try with the following then:

```ruby
uid: "uid-#{index}"
```